sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -382,59 +382,62 @@ def _create_pipeline_workflow(self, args, pipeline, op_transformers=None, pipeli
+            } for p in pipeline_template['spec']['params']
+            ],
+            'pipelineRef': {
+              'name': pipeline_template['metadata']['name']
+            }
           }
         }
-      }
-
 
-      pod_template = {}
-      for task in task_refs:
-        op = pipeline.ops.get(task['name'])
-        if op.affinity:
-          pod_template['affinity'] = convert_k8s_obj_to_json(op.affinity)
-        if op.tolerations:
-          pod_template['tolerations'] = pod_template.get('tolerations', []) + op.tolerations
-        if op.node_selector:
-          pod_template['nodeSelector'] = op.node_selector
-
-      if pod_template:
-        pipelinerun['spec']['podtemplate'] = pod_template
-
-      # add workflow level timeout to pipeline run
-      if pipeline_conf.timeout:
-        pipelinerun['spec']['timeout'] = '%ds' % pipeline_conf.timeout
-
-      # generate the Tekton service account template
-      service_template = {}
-      if len(pipeline_conf.image_pull_secrets) > 0:
-        service_template = {
-          'apiVersion': 'v1',
-          'kind': 'ServiceAccount',
-          'metadata': {'name': pipelinerun['metadata']['name'] + '-sa'}
-        }
-      for image_pull_secret in pipeline_conf.image_pull_secrets:
-        service_template['imagePullSecrets'] = [{'name': image_pull_secret.name}]
+        pod_template = {}
+        for task in task_refs:
+          op = pipeline.ops.get(task['name'])
+          if op.affinity:
+            pod_template['affinity'] = convert_k8s_obj_to_json(op.affinity)
+          if op.tolerations:
+            pod_template['tolerations'] = pod_template.get('tolerations', []) + op.tolerations
+          if op.node_selector:
+            pod_template['nodeSelector'] = op.node_selector
+
+        if pod_template:
+          pipelinerun['spec']['podtemplate'] = pod_template
+
+        # add workflow level timeout to pipeline run
+        if pipeline_conf.timeout:
+          pipelinerun['spec']['timeout'] = '%ds' % pipeline_conf.timeout
+
+        # generate the Tekton service account template
+        service_template = {}
+        if len(pipeline_conf.image_pull_secrets) > 0:
+          service_template = {
+            'apiVersion': 'v1',
+            'kind': 'ServiceAccount',
+            'metadata': {'name': pipelinerun['metadata']['name'] + '-sa'}
+          }
+        for image_pull_secret in pipeline_conf.image_pull_secrets:
+          service_template['imagePullSecrets'] = [{'name': image_pull_secret.name}]
 
-      if service_template:
-        workflow = workflow + [service_template]
-        pipelinerun['spec']['serviceAccountName'] = service_template['metadata']['name']
+        if service_template:
+          workflow = workflow + [service_template]
+          pipelinerun['spec']['serviceAccountName'] = service_template['metadata']['name']
 
-      workflow = workflow + [pipelinerun]
+        workflow = workflow + [pipelinerun]
+    except:
+      pass # Intentionally do nothing
```

Comment:
Can you add a one line comment for why we need to pass the errors for no pipeline decorator?